include/jemalloc/internal/prof_structs.h

```diff
@@ -201,4 +208,14 @@ struct prof_tdata_s {
 };
 typedef rb_tree(prof_tdata_t) prof_tdata_tree_t;
 
+struct prof_recent_s {
+	prof_recent_t *next;
+	size_t usize;
+	uint64_t alloc_time;
```

Comment:
Switch to `nstime_t`. Also given that we may have many of this active, reorder the fields in this to minimize the gap in between.

Reply:
Put the time fields at the front. Also reordered `prof_info_t`.